### PR TITLE
debounce, AM page cleanup, data catalog to match

### PR DESCRIPTION
**Pull request by FBRTMaka**
@DanielJMaher , @dlangston , @birdage 

Please review when you have a chance.
screen shot 2015-08-04 at 11 50 07 am

@lukecampbell thanks for the styling review!

### CODE REVIEW

**Comment by DanielJMaher** [comment]
Data Catalog Matches? @FBRTMaka

**Comment by FBRTMaka** [comment]
@DanielJMaher , yeah - matches the header and such:
screen shot 2015-08-04 at 2 49 38 pm

**Comment by DanielJMaher** [comment]
Got it.  Wasnt sure what you were ttalking about.

**Comment by DanielJMaher** [comment]
@birdage Your call.

File: ooiui/templates/asset_management/assetslist.html [hunk 1]
```diff
@@ -159,17 +155,14 @@
...
+        $('#search').val('').focus();
         $(this).hide();
         updateCollection( showAssets);
     });
-    $('#asset-search').keyup(function(e) {
-        $('#search-clear').toggle(Boolean($(this).val()));
-        $('#search-clear').toggle(Boolean($('#asset-search').val()));
-        updateCollection( showAssets );
-    });
+    $('#search').keyup( _.debounce(textInput, 250) );
```
Comment (birdage) [inline ooiui/templates/asset_management/assetslist.html]: Love it!

File: ooiui/templates/asset_management/assetslist.html [hunk 2]
```diff
@@ -177,6 +170,12 @@
     $('#footer').append(footer.el);
 });
 
+function textInput() {
+        $('#search-clear').toggle(Boolean($(this).val()));
```
Comment (birdage) [inline ooiui/templates/asset_management/assetslist.html]: can we use the view here and `this.$el.find(...)`

**Comment by birdage** [comment]
@FBRTMaka we need to move some of the static link `$(#)` to `this.$el.find(...)` inside of the appropriate views, i.e `this.views.<insertviewname>.$el.find(...)`

File: ooiui/templates/asset_management/assetslist.html (hunk 2 above)
Comment (FBRTMaka) [inline ooiui/templates/asset_management/assetslist.html]: negative, this is not part of a view - so you can't use 'this.$el.find(...)'

**Comment by FBRTMaka** [comment]
@birdage I'll look into putting this controller logic in the view - I tend to have reservations whenever I do that, but this could be prudent.

**Comment by FBRTMaka** [comment]
...it is a highly specific event that will likely not need to be super modular.

**Comment by FBRTMaka** [comment]
@birdage yeah you're right, that view does have that element in it's partial so it def needs to be there.

**Comment by FBRTMaka** [comment]
...thaaaat said....that function is just for the debounce wrapper.  It's really a controller for calling the collection update and to re render the views. (MVC!!!)

**Comment by birdage** [comment]
@FBRTMaka i will defer to you on this. let me know when ready...

**Comment by DanielJMaher** [comment]
@birdage He shouldnt merge his own though

**Comment by FBRTMaka** [comment]
I do like keeping scope consistent, but the mvc style I went at this was that that the html page was the controller.  Moving these into  a view ( I did give it a shot ) totally means  complete refactor.

Also, this.$el.find() is behaviorally identical to $() 

From Backbone source:
```
$: function(selector) {
  return this.$el.find(selector);
}
```

Either way, I'm ready to go on it!

**Comment by birdage** [comment]
@DanielJMaher im not asking him to?

@FBRTMaka sometimes its difficult to do it, i understand! good checking tho!

**Comment by DanielJMaher** [comment]
@birdage Sorry.  I thought you meant his deferral on merge. my mistake.  lost context.

**Pull request merged**